src/components/interactive-builder/modals/question/question-form/common/sortable-tag/sortable-tag.scss

```diff
@@ -0,0 +1,43 @@
+@use '@carbon/colors';
+
+.sortableTag {
+  transition: transform 250ms ease;
+  opacity: 1;
+}
+
+.sortableTag.dragging {
+  opacity: 0.5;
+}
+
+.sortableTagWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
```

Comment:
It'll be better if we can highlight the incorrect answer a bit more, like so:
<img width="809" alt="Screenshot 2025-05-19 at 2 24 32 PM" src="https://github.com/user-attachments/assets/852490ae-c710-40bf-a5d8-3a56ad07b1d5" />
To get this, I added some padding to the sortableTagWrapper, and added the missing `invalid` and `invalidIcon` styles.
```suggestion
  width: 100%;
  padding: 0 $layout.spacing-04 0 $layout.spacing-02;
```